src/main/resources/init_db.sql

```diff
@@ -0,0 +1,7 @@
+CREATE SCHEMA IF NOT EXISTS JDBC_INTRO;
+USE JDBC_INTRO;
+CREATE TABLE IF NOT EXISTS books (
+    id BIGINT AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255),
+    price INT
```

Comment:
Why price is INT what if cost will be 9.99? Isnt its better to have same types as price in Book model like DECIMAL(10,2) -10 numbers after , and 2 before

Reply:
Great, thank you!